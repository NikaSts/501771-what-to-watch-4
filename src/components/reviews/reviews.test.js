import React from 'react';
import renderer from 'react-test-renderer';
import Reviews from './reviews.jsx';


it(`Reviews should render correctly`, () => {
  const tree = renderer
    .create(
        <Reviews
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
