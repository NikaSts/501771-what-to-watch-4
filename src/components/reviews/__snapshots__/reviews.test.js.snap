// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews should render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kendall
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8.7
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
